docs/src/components/header/ThemeModeToggle.tsx

```diff
@@ -1,18 +1,62 @@
 import * as React from 'react';
+import { useColorScheme, useTheme } from '@mui/material/styles';
 import IconButton from '@mui/material/IconButton';
 import Tooltip from '@mui/material/Tooltip';
 import DarkModeOutlined from '@mui/icons-material/DarkModeOutlined';
 import LightModeOutlined from '@mui/icons-material/LightModeOutlined';
 
-interface ThemeModeToggleProps {
+const CssVarsModeToggle = (props: { onChange: (checked: boolean) => void }) => {
```

Comment:
To avoid merge conflicts with https://github.com/mui/material-ui/pull/34644?

```suggestion
function CssVarsModeToggle(props: { onChange: (checked: boolean) => void }) {
```

Reply:
Updated.